src/reference/antora/modules/ROOT/pages/null-safety.adoc

```diff
@@ -0,0 +1,6 @@
+[[null-safety]]
+= Null Safety
+
+The Spring Integration codebase is annotated with JSpecify annotations to declare the nullability of its APIs, fields, and related type usages, following Spring Framework's efforts to add https://spring.io/blog/2025/03/10/null-safety-in-spring-apps-with-jspecify-and-null-away[null safety] to the platform. Reading Spring Framework's https://docs.spring.io/spring-framework/reference/7.0/core/null-safety.html#null-safety-guidelines-nullaway[Null-safety reference documentation] is highly recommended to get familiar with these annotations and their semantics.
+
+The primary goal of this null-safety arrangement is to prevent NullPointerExceptions from being thrown at runtime through build-time checks and to use explicit nullability as a way to express the possible absence of a value.
```

Comment:
No need to pluralize `NullPointerException`.
Plus, it has to as a code snippet.